rel/mim1.vars-toml.config

```
@@ -35,6 +35,10 @@
 
   [host_config.modules.mod_disco]
 
+  {{#mod_register}}
+  [host_config.modules.mod_register]
+  {{{mod_register}}}
+  {{/mod_register}}
```

This section does not do anything.

Explanation: there is no `mod_register` variable anywhere, so it will always be hidden. For static domains we just specify the module as always enabled, see `mongooseim.toml`. The tests pass because they ensure that this module is running. I would recommend just including it always, same as for static domains, for consistency. See my suggestion:

```suggestion
  [host_config.modules.mod_register]
  
```